CHANGES.md

```diff
@@ -69,6 +69,7 @@ Release Notes.
 * Improve the speed of writing TiDB by batching the SQL execution.
 * Fix wrong service name when IP is node IP in `k8s-mesh`.
 * Support dynamic configurations for openAPI endpoint name grouping rule.
+* Add component definition for `Alibaba Druid`.
```

Comment:
FYI @VictorZeng This is the changelog for main repo. From SkyWalking backend perspective, we don't care about the plugin added at agent side.

Reply:
OK, I get it now.